spec/unit/dsl/plugin_spec.rb

```diff
@@ -512,7 +512,7 @@
     end
 
     it "saves platforms across multiple plugins" do
-      plugin = Ohai.plugin(:Test) { collect_data {} }
+      Ohai.plugin(:Test) { collect_data {} }
```

Comment:
I think this line can just be removed.

Reply:
Nope, @tpowell-progress pointed out this is intentional, but we should make a comment...
```suggestion
      # run ohai once for _all_ plugins
      Ohai.plugin(:Test) { collect_data {} }
      # then run just 2 plugins, and make sure the other plugins' data is preserved.
```